src/mmw/js/src/core/views.js

```diff
@@ -265,11 +265,17 @@ var MapView = Marionette.ItemView.extend({
                 maximumAge : maxAge,
                 timeout : timeout
             };
-            navigator.geolocation.getCurrentPosition(
-                geolocation_success,
-                _.noop,
-                geolocationOptions
-            );
+
+            // Wait a bit and then get the location. Suppresses issues on Safari
+            // in which immediately requesting the location results in an
+            // infinite request loop for geolocation permissions.
+            setTimeout(function() {
+                navigator.geolocation.getCurrentPosition(
+                    geolocation_success,
+                    _.noop,
+                    geolocationOptions
+                );
+            }, 500);
```

Comment:
I'm not against this but it's not really and important part of the application that I think we need to tune. We just need some delay and I feel like adding a variable for it just adds a little more indirection without any real gain.